sgl-kernel/setup_rocm.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2025 SGLang Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import multiprocessing
+import os
+import sys
+from pathlib import Path
+
+import torch
+from setuptools import find_packages, setup
+from torch.utils.cpp_extension import BuildExtension, CUDAExtension
+
+root = Path(__file__).parent.resolve()
+
+if "bdist_wheel" in sys.argv and "--plat-name" not in sys.argv:
+    sys.argv.extend(["--plat-name", "manylinux2014_x86_64"])
+
+
+def _get_version():
+    with open(root / "pyproject.toml") as f:
+        for line in f:
+            if line.startswith("version"):
+                return line.split("=")[1].strip().strip('"')
+
+
+operator_namespace = "sgl_kernels"
+include_dirs = [
+    root / "src" / "sgl-kernel" / "include",
+    root / "src" / "sgl-kernel" / "csrc",
+]
+
+sources = [
+    "src/sgl-kernel/torch_extension_rocm.cc",
+    "src/sgl-kernel/csrc/moe_align_kernel.cu",
```

Comment:
Do you need to update this for ROCm?
https://github.com/sgl-project/sglang/blob/70817a7eae0055cb3c98c7827b73b8058fb342f4/sgl-kernel/src/sgl-kernel/csrc/moe_align_kernel.cu#L27-L28

Reply:
Not this moment, works now. We will do deeper custom later.

Reply:
ok I just remember to remove the ROCm-related code earlier.

Reply:
@zhyncs created 2 issues to work on asap, thx.